modules/hcp-ibm-power-create-heterogeneous-nodepools-agent-hc.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-ibm-power-create-heterogeneous-nodepools-agent-hc_{context}"]
+= Create Heterogeneous NodePools on Agent HostedClusters
+
+This explains how to create heterogeneous nodepools on an agent platform. Please refer to set up the environment using Agent Platform from section 4.6.1, this section only covers the things needed to configure to have heterogeneous nodepools.
+
+.Procedure
+
+. Configure AgentServiceConfig with two heterogeneous architecture OS images:
++
+[source,terminal]
+----
+$ export DB_VOLUME_SIZE="10Gi"
+$ export FS_VOLUME_SIZE="10Gi"
+$ export OCP_VERSION="4.18.0"
+$ export OCP_MAJMIN=${OCP_VERSION%.*}
+$ export ARCH_X86="x86_64"
+$ export OCP_RELEASE_VERSION_X86=$(curl -s https://mirror.openshift.com/pub/openshift-v4/${ARCH_X86}/clients/ocp/${OCP_VERSION}/release.txt | awk '/machine-os / { print $2 }')
+$ export ISO_URL_X86="https://mirror.openshift.com/pub/openshift-v4/${ARCH_X86}/dependencies/rhcos/${OCP_MAJMIN}/${OCP_VERSION}/rhcos-${OCP_VERSION}-${ARCH_X86}-live.${ARCH_X86}.iso"
+$ export ROOT_FS_URL_X8="https://mirror.openshift.com/pub/openshift-v4/${ARCH_X86}/dependencies/rhcos/${OCP_MAJMIN}/${OCP_VERSION}/rhcos-${OCP_VERSION}-${ARCH_X86}-live-rootfs.${ARCH_X86}.img"
+$ export ARCH_PPC64LE="ppc64le"
+$ export OCP_RELEASE_VERSION_PPC64LE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/${ARCH_PPC64LE}/clients/ocp/${OCP_VERSION}/release.txt | awk '/machine-os / { print $2 }')
+$ export ISO_URL_PPC64LE="https://mirror.openshift.com/pub/openshift-v4/${ARCH_PPC64LE}/dependencies/rhcos/${OCP_MAJMIN}/${OCP_VERSION}/rhcos-${OCP_VERSION}-${ARCH_PPC64LE}-live.${ARCH_PPC64LE}.iso"
+$ export ROOT_FS_URL_PPC64LE="https://mirror.openshift.com/pub/openshift-v4/${ARCH_PPC64LE}/dependencies/rhcos/${OCP_MAJMIN}/${OCP_VERSION}/rhcos-${OCP_VERSION}-${ARCH_PPC64LE}-live-rootfs.${ARCH_PPC64LE}.img"
+----
++
```

Comment:
In the following code block and others like it in this file, the code needs to be rewritten to separate out the content that the user will enter in the YAML file and the command to apply the changes. For an example, see steps 1 - 3 in [Creating an InfraEnv resource for hosted control planes on IBM Z](https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/hosted_control_planes/deploying-hosted-control-planes#hcp-ibm-z-infraenv_hcp-deploy-ibmz).